packages/ui/src/styles/globals.css

```
@@ -1,123 +1,127 @@
-    --chart-5: 340 75% 55%;
-  }
-}
-
-html {
-  scroll-behavior: smooth;
+@import "tailwindcss";
+@source "../../../../apps/www/**/*.{ts,tsx}";
+@source "../../../../apps/dash/**/*.{ts,tsx}";
+@source "../**/*.{ts,tsx}";
+
+@import "tw-animate-css";
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)